src/Microsoft.DocAsCode.Build.ManagedReference/BuildManagedReferenceDocument.cs

```diff
@@ -22,38 +17,25 @@ public class BuildManagedReferenceDocument : BuildReferenceDocumentBase, ISuppor
             new CompositeModelAttributeHandler(
                 new UniqueIdentityReferenceHandler(),
                 new MarkdownContentHandler()
-                );
+            );
```

Comment:
what if child has different handlers? Or make the handlers and context as default?

---
In reply to: [114489466](https://github.com/dotnet/docfx/pull/1621#discussion_r114489466) [](ancestors = 114489466)